lib/components/app/app-menu.js

```diff
@@ -52,47 +142,69 @@ class AppMenu extends Component {
       toggleMailables
     } = this.props
 
+    const { isPaneOpen } = this.state
     return (
-      <div aria-label='app-menu' className='app-menu' role='navigation'>
-        <DropdownButton
-          aria-label='Application Menu'
-          className='app-menu-button'
-          id='app-menu'
-          noCaret
-          title={(<Icon type='bars' />)}>
-          <MenuItem onClick={this._showRouteViewer}>
-            <Icon type='bus' /> {languageConfig.routeViewer || 'Route Viewer'}
-          </MenuItem>
-          {callTakerEnabled &&
-            <MenuItem onClick={resetAndToggleCallHistory}>
-              <Icon type='history' /> Call History
-            </MenuItem>
-          }
-          {fieldTripEnabled &&
-            <MenuItem onClick={resetAndToggleFieldTrips}>
-              <Icon type='graduation-cap' /> Field Trip
+      <>
+        <div
+          aria-label={isPaneOpen ? 'Close Menu' : 'Open Menu'}
+          className='app-menu-icon'
+          onClick={this._expandPane}
+          onKeyDown={this._expandPane}
+          role='button'
+          tabIndex={0}>
```

Comment:
Move `>` to new line.

Reply:
Prettier run in 542d756098ceae9ad56f74334fae242b4be7c938